src/components/MyProfile.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import './css/MyProfile.css';
+
+function MyProfile() {
+  /* eslint-disable */ 
+  const reservedRockets = useSelector((state) => state.rocketReducers.filter((rocket) => rocket.rocketBooked));
+  const reservedMissions = useSelector((state) => state.missionReducers.filter((mission) => mission.missionJoined));
+  /* eslint-enable */
```

Comment:
- [ ] [OPTIONAL] Kindly remove the disable linter and try to fix the linter errors. remember than linter help you to write a clean code and more professional. Please Tean do that and let's make your project looks professional.